e2e-tests/specs/relay-premium-upgrade.spec.ts

```diff
@@ -5,6 +5,8 @@ import { checkAuthState, defaultScreenshotOpts } from '../e2eTestUtils/helpers';
 test.use({ storageState: 'state.json' })
 test.describe.configure({ mode: 'parallel' });
 test.describe('Premium Relay - Purchase Premium Flow, Desktop', () => {
+  test.skip(({ browserName }) => browserName === 'firefox', 'FxA Issue with with authenication');
```

Comment:
also, if we have a jira ticket for this, we should add it here of in a comment/annotation.

```suggestion
  test.skip(({ browserName }) => browserName === 'firefox', 'FxA Issue with with authentication');
```